shared/fs/banner/conflict-banner-container.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as React from 'react'
+import * as Constants from '../../constants/fs'
+import * as Types from '../../constants/types/fs'
+import {namedConnect} from '../../util/container'
+import ConflictBanner, {getHeight} from './conflict-banner'
+import * as RowTypes from '../row/types'
+import openUrl from '../../util/open-url'
+
+type OwnProps = {|path: Types.Path, conflictState: Types.ConflictState|}
+
+const mapStateToProps = (state, {path}: OwnProps) => ({
+  _tlf: Constants.getTlfFromPath(state.fs.tlfs, path),
+})
+
+const mapDispatchToProps = dispatch => ({
+  onFeedback: () => {},
+  onFinishResolving: () => {},
+  onHelp: () => openUrl('https://keybase.io/docs/kbfs/understanding_kbfs#conflict_resolution'),
+  onSeeOtherView: () => {},
+  onStartResolving: () => {},
+})
+
+const mergeProps = (s, d, o) => ({
+  ...s,
```

Comment:
I don't think you need `...s` here.

Reply:
I need it for the `conflictState`. Though I might move collecting that into the container now that this is not going to be a row anymore..